### PR TITLE
Revisions

### PR DESCRIPTION
**Pull request by jdegoes**
Many much more substantial changes and miscellaneous cleanup.

### CODE REVIEW

**Comment by jdegoes** [comment]
@ankushnarula throwaway1973 @FaustDeGoethe Feel free to review.

File: COC.md [hunk 1]
```diff
@@ -57,13 +56,13 @@ These requirements on behavior apply to _members_ at all times, even when they a
 
 **No Victimless Violations**. If you believe you have been adversely affected by a _violation_ but are not _aggrieved_, you may choose to not report the _incident_. In this case, we will not treat the incident as a violation.
 
-**Reporting Process**. _Active participation_ incidents must be reported to us within 15 days by _aggrieved_ members, and may not be reported by third-parties. Inactive Participation incidents may be reported to us within 5 years of their occurrence, and by anyone, even non-_members_.
+**Reporting Process**. _Active participation_ incidents must be reported to us within 15 days by _aggrieved_ members, and may not be reported by third-parties. _Inactive participation_ incidents may be reported to us within 5 years of their occurrence, and by anyone, even non-_members_.
```
Comment (FaustDeGoethe) [inline COC.md]: > and by anyone, even non-_members_.

Other ways I could see phrasing it:

by anyone, regardless of membership status.
by members and non-members alike.

File: COC.md [hunk 2]
```diff
@@ -57,13 +56,13 @@ These requirements on behavior apply to _members_ at all times, even when they a
...
 
-**Unofficial Resolution**. For minor offenses and in cases where they prefer doing so, we encourage _aggrieved_ members to speak to _accused_ members, using the language of non-violent communication (NVC). If you would like to do this with the help of an independent mediator, contact us and we will arrange for one.
+**Unofficial Resolution**. For minor offenses and only in cases where they prefer doing so, we encourage _aggrieved_ members to speak to _accused_ members directly, using the language of non-violent communication (NVC). If you would like to do this with the help of an independent mediator, we will arrange for one.
```
Comment (FaustDeGoethe) [inline COC.md]: For minor offences**,** and only ...

File: COC.md [hunk 3]
```diff
@@ -57,13 +56,13 @@ These requirements on behavior apply to _members_ at all times, even when they a
...
 
 **Official Resolution**. If you want an official intervention, we will appoint an arbiter (consisting of one or more persons). The arbiter will speak individually to all parties, including witnesses, before deciding on a course of action, which will involve rejecting the reported violation, or accepting it and imposing a consequence on the _accused_ member.
 
-**Consequences**. _Violators_ may be warned, asked to apologize or undergo training, counseling, or mediation as a condition of continued participation, or ejected and banned from the _community_, at the sole discretion of the arbiter. In no case will the consequence for violating FCOP exceed banishment, unless the violation is also governed by a separate contractual agreement or local law.
+**Consequences**. _Violators_ may be warned, asked to apologize or undergo training, counseling, or mediation as a condition of continued participation, or ejected and banned from the _community_, at the sole discretion of the arbiter. In no case will the consequence for violating FCOP exceed banishment, unless the violation is also governed by separate contractual agreements or local laws.
```
Comment (FaustDeGoethe) [inline COC.md]: > ... participation, or ejected and banned from the _community_, at the sole discretion of the arbiter.
They may also be ejected or banned from the community, ....

File: COC.md (hunk 1 above)
Comment (FaustDeGoethe) [inline COC.md]: > and by anyone, even non-_members_.

Other ways I could see phrasing it:

by anyone, regardless of membership status.
by members and non-members alike.

File: COC.md (hunk 2 above)
Comment (FaustDeGoethe) [inline COC.md]: For minor offenses**,** and only ...

File: COC.md [hunk 4]
```diff
@@ -79,25 +78,34 @@ In the event there is a dispute about the meaning of any term or clause in FCOP,
...
-    * **Criminal Offenses**. Any illegal behavior considered a criminal offense in the country of jurisdiction.
-    * **Community Sabotage**. Any behavior (excluding non-violent communication) directed at sabotaging the community for political, religious, ideological, or moral reasons.
-    * **Professional Sabotage**. Any behavior directed at sabotaging a _member_'s career for political, religious, ideological, or moral reasons; including attempting to no-platform a _member_ or pressuring an employer to fire a _member_.
+    * **Criminal Offenses**. Any illegal behavior considered a criminal offense in the community's country of jurisdiction.
+    * **Community Sabotage**. Any behavior directed at sabotaging the community.
+    * **Professional Sabotage**. Any behavior directed at sabotaging a _member_'s career for any reason that is unrelated to the _member_'s performance in their profession, including but not limited to attempting to no-platform a _member_ or pressuring an employer to fire a _member_, or encouraging others to do the same.
```
Comment (FaustDeGoethe) [inline COC.md]: > attempting to no-platform a _member_ or pressuring an employer to fire a _member_, or encouraging others to do the same.

attempting to no-platform a _member_, pressuring an employer to fire a _member_, or encouraging others to do the same.

File: COC.md [hunk 5]
```diff
@@ -79,25 +78,34 @@ In the event there is a dispute about the meaning of any term or clause in FCOP,
...
+ * **Denigration**. We define _denigration_ as the _communication_ of the idea that any physical or mental characteristic of any person is superior or inferior to any other. This explicitly excludes communication of the idea that objective, falsifiable statements espoused by someone are inconsistent or unsupported or contradicted by data.
  * **Discrimination**. We define _discrimination_ as any favoritism shown or withheld to someone either on the basis of a stereotype or a non-community related group membership.
- * **Harassment**. We define _harassment_ as an _interaction_ with someone who does not consent to the interaction.
+ * **Doxxing** We define _doxxing_ to be the unauthorized dissemination of any details about _members_ learned during _active participation_, including but not limited to real name, address, phone number, or photo identity.
```
Comment (FaustDeGoethe) [inline COC.md]: > of any details

any personal information

File: COC.md [hunk 6]
```diff
@@ -79,25 +78,34 @@ In the event there is a dispute about the meaning of any term or clause in FCOP,
...
  * **Incident**. We define _incident_ as a possible infringement of the FCOP.
  * **Interaction**. We define _interaction_ as any one-on-one communication, physical contact (with person or property), close proximity, or persistent gaze. Interaction does not include opt-in, broadcast-based communication within the Community.
  * **Members**. We define a _member_ of the community to be any civil individual who we allow to actively participate in the community.
- * **Stereotyping**. We define _stereotyping_ as behavior that infers or implies one characteristic of an individual based on their [perceived] membership in some group or category.
+ * **Moralizing**. We define _moralizing_ as an attempt to evangelize one's moral system or values to others, whether by communicating the idea that their moral system or values are superior to others, or by intimidating people to profess or practice their moral system or values.
+ * **Shaming**. We define _shaming_ to be unauthorized, negative communication to non-_members_ about a _member_'s behavior inside the _community_, where the discussed _member_ decides what is negative.
```
Comment (FaustDeGoethe) [inline COC.md]: > where the discussed _member_ decides what is negative.

where "negative" is defined by the _member_ on a case by case basis.

File: COC.md [hunk 7]
```diff
@@ -79,25 +78,34 @@ In the event there is a dispute about the meaning of any term or clause in FCOP,
...
+ * **Stereotyping**. We define _stereotyping_ as _communication_ that implies one characteristic of an individual based on their perceived membership in some (unrelated or loosely related) group or category.
  * **Violation**. We define _violation_ as an actual infringement of the FCOP as determined by an official arbiter.
- * **Violator**. We define _violator_ as an _accused_ member who has been judged to have broken the terms and conditions of FCOP.
+ * **Violator**. We define _violator_ as an _accused_ member who the arbiter has decided has broken the terms and conditions of FCOP.
```
Comment (FaustDeGoethe) [inline COC.md]: who, the arbiter has decided, has ...

or

who has broken the terms and conditions of FCOP (as determined by an _arbiter_).

**Pull request merged**